Maestro/subscriptions.json

```diff
@@ -1069,6 +1071,7 @@
       "triggerPaths": [
         "https://github.com/aspnet/SignalR-Client-Cpp/blob/release/**/*",
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
+        "https://github.com/aspnet/AspNetWebHooks/blob/release/**/*",
```

Comment:
Always find it strange the `github-dnceng-azdo-internal-merge-mirror` task takes a list of GitHub repos when it mostly operates on AzDO repos. Does it actually merge from the GitHub branches rather than the AzDO public one❔

Reply:
It merges from the public branch into the `internal/` branch so that internal servicing builds have all the public changes.